### PR TITLE
[4] Add drivers locations commands , entity, persistence and tests.

### PR DESCRIPTION
**Pull request by Yurchenko-Anton**
Add new constraint in drivers locations table. And moved some classes to another package

### CODE REVIEW

File: src/main/resources/db.changelog/10-create-drivers-locations-table.xml [hunk 1]
```diff
@@ -11,7 +11,9 @@
             <column name="id" type="int">
                 <constraints nullable="false" primaryKey="true"/>
             </column>
-            <column name="driver_id" type="int"/>
+            <column name="driver_id" type="int">
```
Comment (asinchuk) [inline src/main/resources/db.changelog/10-create-drivers-locations-table.xml:14]: it's a super bad practice, please create new changeset

File: src/main/resources/changelog-master.xml [hunk 2]
```diff
@@ -8,5 +8,6 @@
     <include file="db.changelog/01-create-locations-table.xml" relativeToChangelogFile="true"/>
     <include file="db.changelog/01-populate-locations-table.xml" relativeToChangelogFile="true"/>
     <include file="db.changelog/10-create-drivers-locations-table.xml" relativeToChangelogFile="true"/>
+    <include file="db.changelog/10-update-drivers-locations-table.xml" relativeToChangelogFile="true"/>
```
Comment (asinchuk) [inline src/main/resources/changelog-master.xml:11]: It's a non informative naming, please change to more precise name, it should reflect exact change

File: src/main/resources/db.changelog/10-create-drivers-locations-table.xml [hunk 3]
```diff
@@ -11,9 +11,7 @@
             <column name="id" type="int">
                 <constraints nullable="false" primaryKey="true"/>
             </column>
-            <column name="driver_id" type="int">
-                <constraints unique="true"/>
-            </column>
+            <column name="driver_id" type="int"/>
```
Comment (asinchuk) [inline src/main/resources/db.changelog/10-create-drivers-locations-table.xml:14]: I already told you don't touch merged changesets

Comment (asinchuk) [inline src/main/resources/db.changelog/10-create-drivers-locations-table.xml:14]: You can create in advance PR's with DB changesets and then PR with code that rely on them

File: src/main/resources/db.changelog/10-update-drivers-locations-table.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="7" author="ayurchenko">
```
Comment (asinchuk) [inline src/main/resources/db.changelog/10-update-drivers-locations-table.xml:9]: Please avoid cross id

File: src/main/resources/db.changelog/10-create-drivers-locations-table.xml (hunk 3 above)
Comment (Yurchenko-Anton) [reply src/main/resources/db.changelog/10-create-drivers-locations-table.xml:14]: return old changeset

File: src/main/resources/db.changelog/01-create-locations-table.xml [hunk 5]
```diff
@@ -5,7 +5,7 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="4" author="ayurchenko">
```
Comment (asinchuk) [inline src/main/resources/db.changelog/01-create-locations-table.xml:8]: merged changesets shouldn't be touched, they are immutable. Maybe introduce id range is better idea? Like 1-9999 for security, 10000 - 19999 for orders and 20000 - 29999 for matcher?

**Review by asinchuk — APPROVED** [review]

**Pull request merged**